app/controllers/status_controller.rb

```diff
@@ -16,13 +17,14 @@ def index
   # configuring proxies or load balancers.
   def whoami
     audit_success
-
     render json: {
       client_ip: request.ip,
       user_agent: request.user_agent,
       account: token_user.account,
       username: token_user.login,
-      token_issued_at: Time.at(token_user.token.claims["iat"])
+      # This is here to fix a bug that occurs only in test
+      # We don't fully understand what causes the bug but this is the hack we settled on to fix it.
+      token_issued_at: ((defined? token_user.token.claims) == nil) ? nil : Time.at(token_user.token.claims["iat"])
```

Comment:
can't we write 
```
token_issued_at: token_user.token.claims.nil? ? nil : Time.at(token_user.token.claims["iat"])
```

Reply:
can you elaborate a bit on the bug?

Reply:
The method '**claims**' doesn't exist when I run the test. I tried to debug it and change the token on the sender side (rest_helpers.rb in cucumber and other places using Slosilo). but, I couldn't find the cause.

Reply:
please look into it some more. it is best not to change application code for the purposes of tests.